tests/integration/submission/test_upload.py

```diff
@@ -447,47 +527,99 @@ def test_dont_enforce_file_hash_size_uniqueness(
     assert indexd_client.get(new_id).urls == [new_url]
 
 
-def test_is_updatable_file(client, pg_driver, indexd_client):
-    """Test _is_updatable_file_node method
+def test_cannot_update_node_in_state_submitted(
+        client, pg_driver, indexd_client, submitter, cgci_blgsp):
+    """Verify that indexd document is not updated after the node was submitted
     """
+    _, sur_entity = data_file_creation(
+        client, submitter, sur_filename='submitted_unaligned_reads.json')
 
-    did = 'bef870b0-1d2a-4873-b0db-14994b2f89bd'
-    url = '/some/url'
+    node_id = sur_entity['id']
+    doc_original = indexd_client.get(node_id)
 
-    # Create dummy file node and corresponding indexd record
-    node = SubmittedAlignedReads(did)
-    indexd_client.create(
-        did=did,
-        urls=[url],
-        hashes={'md5': '0'*32},
-        size=1,
-    )
-    ALLOWED_STATES = [
-        'registered',
-        'uploading',
-        'uploaded',
-        'validating',
-    ]
+    # Set node state to 'submitted'
+    with pg_driver.session_scope():
+        sur_node = pg_driver.nodes().get(node_id)
+        sur_node.state = 'submitted'
 
-    DISALLOWED_STATES = [
-        'validated',
-    ]
-    transaction = MagicMock()
-    transaction.indexd = indexd_client
-    entity = FileUploadEntity(transaction)
-    entity.s3_url = url
+    new_version = read_json_data(
+        os.path.join(DATA_DIR, 'submitted_unaligned_reads_new.json'))
 
-    for file_state in ALLOWED_STATES:
-        # set node's url state in indexd
-        indexd_doc = indexd_client.get(did)
-        set_indexd_state(indexd_doc, url, file_state)
+    # Actual update call - should fail
+    resp = client.put(BLGSP_PATH, headers=submitter,
+                      data=json.dumps(new_version))
+    doc_after_put = indexd_client.get(node_id)
 
-        # check if updatable
-        assert entity.is_updatable_file_node(node)
+    assert resp.status_code == 400
+    assert doc_original.to_json() == doc_after_put.to_json()
 
-    for file_state in DISALLOWED_STATES:
-        # set node's url state in indexd
-        indexd_doc = indexd_client.get(did)
-        set_indexd_state(indexd_doc, url, file_state)
-        # check if not updatable
-        assert not entity.is_updatable_file_node(node)
+
+@pytest.mark.config_toggle(
+    parameters={
+        'CREATE_REPLACEABLE': True,
+    }
+)
+def test_update_multiple_one_fails(
```

Comment:
one big ole unit test. 

Any functionality in here that could be reused in other tests? Then push to a fixture I feel like the mark.config_toggle might be better served as the flask config from pytest to wrap all the fixtures together to get to the state where you are twiddling the submitted state/ and data?